<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>299</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>449</width>
    <height>299</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScreenGrab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="scrLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labTypeScr">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Type: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxTypeScr">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Type of screenshot</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Full screen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Window</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Screen area</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="butNew">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Getting new screenshot</string>
        </property>
        <property name="text">
         <string>New Screen</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+N</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labDelay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Delay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="delayBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Delay in seconds before taking screenshot</string>
        </property>
        <property name="suffix">
         <string> sec</string>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butSave">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Save current screenshot into graphical file</string>
        </property>
        <property name="text">
         <string>Save Screen</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butCopy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Copy current screenshot into clipboard</string>
        </property>
        <property name="text">
         <string>Copy Screen</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butUpload">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butOpt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Options dialog</string>
        </property>
        <property name="text">
         <string>Options</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+O</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butHelp">
        <property name="toolTip">
         <string>Show help information</string>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
        <property name="shortcut">
         <string>F1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butInfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Information about ScreenGrab</string>
        </property>
        <property name="text">
         <string>Information</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+I</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butQuit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Exit from ScreenGrab</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+Q</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>delayBox</tabstop>
  <tabstop>butNew</tabstop>
  <tabstop>butSave</tabstop>
  <tabstop>butCopy</tabstop>
  <tabstop>butOpt</tabstop>
  <tabstop>butHelp</tabstop>
  <tabstop>butInfo</tabstop>
  <tabstop>butQuit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
